configure.ac

```diff
@@ -23,10 +23,10 @@
 #       This file is derived from tpm-tool's configure.in.
 #
 
-AC_INIT(swtpm, 0.7.0)
-AC_PREREQ(2.12)
+AC_INIT([swtpm],[0.6.0])
```

Comment:
Not going back to 0.6.0, please.

Reply:
Title of patch has a typo. What was 2.12 'originally'?

Reply:
Commit text typo: chenges -> changes. Can you not squash this patch with the one that removed this stuff previously?